girder/molecules/molecules/utilities/async_requests.py

```diff
@@ -120,3 +124,39 @@ def _finish_3d_coords_gen(inchikey, user, future):
         semantic.upload_molecule(MoleculeModel().findOne(query))
     except requests.ConnectionError:
         print(TerminalColor.warning('WARNING: Couldn\'t connect to Jena.'))
+
+
+def schedule_orbital_gen(cjson, mo, id, orig_mo):
+    cjson['generating_orbital'] = True
+
+    url = 'http://avogadro:5000/calculate'
+    data = {
+        'cjson': cjson,
+        'mo': mo,
+    }
+
+    session = FuturesSession()
+    future = session.post(url, json=data)
+
+    future.add_done_callback(functools.partial(
+        _finish_orbital_gen, mo, id, getCurrentUser(), orig_mo))
```

Comment:
Probably want some error handling here as well, not sure if  requests-future has a `raise_for_status`?